lib/api/bucketDeleteTagging.js

```diff
@@ -0,0 +1,44 @@
+const { waterfall } = require('async');
+
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const metadata = require('../metadata/wrapper');
+
+/**
+ * Bucket Delete Tagging - Delete a bucket's Tagging
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+function bucketDeleteTagging(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'bucketDeleteTagging' });
+
+    const bucketName = request.bucketName;
+    const metadataValParams = {
+        authInfo,
+        bucketName,
+        requestType: 'bucketDeleteTagging',
+    };
+    return waterfall([
```

Comment:
After looking at the issue and discussion with @anurag4DSB , this would require promisifying some elements and will take some time we aren't ready to spend right now given it would be better done in a general code cleanup; will create a ticket.

Reply:
You mean this guy? https://nodejs.org/api/util.html#utilpromisifyoriginal

Reply:
Used async await, let me know what you think https://github.com/scality/cloudserver/pull/4480/commits/13907cb2dd25f0511c3929f5987e111a68487c02